docs/charts/index.md

```diff
@@ -0,0 +1,219 @@
+# Charts Gallery
+
+## Exploratory Analysis
+
+::cards::
+[
+  {
+    "title": "`missingness chart`",
+    "image": "./img/missingness_chart.png",
+    "url": "./missingness_chart.ipynb"
+  },
+  {
+    "title": "`profile columns`",
+    "image": "./img/profile_columns.png",
+    "url": "./profile_columns.ipynb"
+  },
+]
+::/cards::
+
+## Blocking
+
+::cards::
+[
+    {
+    "title": "`cumulative num comparisons from blocking rules chart`",
+    "image": "./img/cumulative_num_comparisons_from_blocking_rules_chart.png",
+    "url": "./cumulative_num_comparisons_from_blocking_rules_chart.ipynb"
+    },
+]
+::/cards::
+
+## Comparison Helpers
+
+::cards::
+[
+    {
+    "title": "`comparator score chart`",
+    "image": "./img/comparator_score_chart.png",
+    "url": "../topic_guides/comparisons/choosing_comparators.ipynb#comparing-string-similarity-and-distance-scores"
+    },
+    {
+    "title": "`comparator score threshold chart`",
+    "image": "./img/comparator_score_threshold_chart.png",
+    "url": "../topic_guides/comparisons/choosing_comparators.ipynb#choosing-thresholds"
+    },
+    {
+    "title": "`phonetic match chart`",
+    "image": "./img/phonetic_match_chart.png",
+    "url": "../topic_guides/comparisons/choosing_comparators.ipynb#phonetic-matching"
```

Comment:
This link works here, but not in the sidebar itself (I'll try to find that now...)

Reply:
Basically, this doesn't work for me...

![Screenshot 2023-08-17 at 10 02 45](https://github.com/moj-analytical-services/splink/assets/45356472/ac580ac6-c7ff-4da6-8022-671a974071e8)

Reply:
fixed